cdap-ui/app/hydrator/leftpanel.less

```diff
@@ -309,9 +309,9 @@ body.theme-cdap.state-hydrator-create {
         }
         div.side-panel.top {
           .hydrator-metadata {
-            width: ~"calc(100vw - 755px)";
-            width: ~"-moz-calc(100vw - 755px)";
-            width: ~"-webkit-calc(100vw - 755px)";
+            width: ~"calc(100vw - 805px)";
```

Comment:
Yes you are right. I remember Rohit fixed this problem when we restyled pipeline actions in last release. This could be because of the extra button that we added in this PR.